src/Backend/Core/Installer/ModuleInstaller.php

```diff
@@ -884,67 +885,70 @@ protected function makeSearchable($module, $searchable = true, $weight = 1)
     /**
      * Set the rights for an action
      *
-     * @param int    $groupId The group wherefor the rights will be set.
-     * @param string $module  The module wherin the action appears.
-     * @param string $action  The action wherefor the rights have to set.
-     * @param int    $level   The leve, default is 7 (max).
+     * @param int|Group     $group       The group wherefor the rights will be set.
```

Comment:
Shouldn't you always send a Group object in? Why keep the int options? It just gives more complexity and is_object checks.